services/Skyra.Database/Skyra.Database.csproj

```diff
@@ -15,4 +15,8 @@
       <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="5.0.2" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Folder Include="Migrations" />
+    </ItemGroup>
```

Comment:
@Tylertron1998 it auto-added this. Should this be here?

Reply:
There is no need for it to be there, no - can be removed safely